Sources/Files.swift

```diff
@@ -298,6 +298,24 @@ public class FileSystem {
         return try! Folder(path: ProcessInfo.processInfo.homeFolderPath, using: fileManager)
     }
     
+    /// A reference to the document folder used by this file system
+    public var documentFolder: Folder? {
+        guard let url = try? fileManager.url(for: .documentDirectory, in: .userDomainMask, appropriateFor: nil, create: false) else { return nil }
+        
+        do {
+            return try Folder(path: url.path, using: fileManager)
+        } catch {
+            return nil
+        }
+    }
+    
+    /// A reference to the library folder used by this file system
+    public var libraryFolder: Folder {
```

Comment:
If it's available on all platforms, then you can just do what I did for `homeFolder`, like this:

```swift
func testAccessingLibraryFolder() {
        XCTAssertNotNil(FileSystem().libraryFolder)
    }
```
Just to make sure it doens't crash.